content/_data/changelogs/weekly.yml

```diff
@@ -7034,6 +7034,89 @@
   # pull: 4596 (PR title: [JENKINS-61409] Websockets: Use AbstractByteBufferCommandTransport to transport messages) combined with pull 4601
   # pull: 4605 (PR title: Do not print random binary data during WebSocketAgentsTest)
 
+- version: 2.230
+  date: 2020-04-04
+  changes:
+  - type: rfe
+    category: rfe
+    pull: 4610
+    issue: 61478
+    authors:
+    - romenrg
+    message: |-
+      Style improvements on alert banners.
+  - type: rfe
+    category: rfe
+    pull: 4575
+    issue: 61467
+    authors:
+    - daniel-beck
+    - timja
+    message: |-
+      Do not show disabled permissions in permission errors when checking for any of several permissions.
+  - type: rfe
+    category: rfe
+    pull: 4616
+    authors:
+    - jglick
+    message: |-
+      Allow hyperlinks to be used when displaying causes of blockage related to labels rather than individual nodes.
+  - type: rfe
+    category: rfe
+    pull: 3947
+    issue: 5597
+    authors:
+    - mwinter69
+    message: |-
+      Add option to configure follow symlinks when archiving artifacts.
+  - type: rfe
+    category: rfe
+    pull: 4572
+    issue: 61453
+    authors:
+    - escoem
+    - timja
+    message: |-
+      <em>Prepare for Shutdown</em> management link is now accessible to users with <code>Overall/Manage</code> permission in addition to the usual <code>Overall/Administer</code>.
+  - type: rfe
+    category: rfe
+    pull: 4599
+    issue: 61496
+    authors:
+    - fqueiruga
+    message: |-
+      Updated footer styles and removed the "Page Generated" information.
```

Comment:
🐛  "Removed the "Page Generated" information." was actually excluded from the final version of the PR